### DISCUSSION TITLE
No check npm devDependencies in group

### DISCUSSION THREAD

**Discussion by suiyun39** [renovatebot/renovate#21437] [Request Help] [answered]
### I'm using the Mend Renovate hosted app on github.com.

- [X] I'm using the Mend Renovate hosted app.

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe your problem.

repo: [babel-plugin-jsx-asset-url-import](https://github.com/NoDocCat/babel-plugin-jsx-asset-url-import)

I create a group for all minor and patch update, if manager is github-actions, it works fine. But when manager is npm, I only see major update in the dashboard.

This is my config:

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base",
    "schedule:weekends",
    ":dependencyDashboardApproval",
    ":assignAndReview(NoDocCat)",
    ":labels(dependencies)"
  ],
  "rebaseWhen": "never",
  "packageRules": [
    {
      "groupName": "{{manager}} dependencies",
      "matchPackagePatterns": [
        "*"
      ],
      "matchUpdateTypes": [
        "minor",
        "patch"
      ]
    }
  ]
}
```

In dashboard, I only see these:

![image](https://user-images.githubusercontent.com/20502666/231369889-c212c6cc-30ac-418a-a76a-e989d2f8a3cb.png)

But, these deps already outdated:

![image](https://user-images.githubusercontent.com/20502666/231369936-ff8c9b4e-5c00-480b-9c41-56f248943791.png)

### Relevant debug logs.

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment]
by default renovate don't bump ranges when the update is in range.

checkout range strategy config option.

**Reply by suiyun39** [reply]
In the repo: [react-vant](https://github.com/3lang3/react-vant), renovate create a pull request update `eslint 8.21.0 -> 8.38.0`.
The link: [#563](https://github.com/3lang3/react-vant/pull/563), The only difference is that I used the group

**Reply by rarkins** [reply]
Yes, that dependency is pinned to exact version rather than a wide range

**Reply by suiyun39** [reply]
> Yes, that dependency is pinned to exact version rather than a wide range

Sorry, i overlooked this. thanks @rarkins and @viceice

**Comment by rarkins** [comment, marked as answer]
Renovate doesn't currently parse pnpm's lock file, so cannot implement rangeStrategy=update-lockfile.

The best solution is to pin dependencies in package.json

**Reply by suiyun39** [reply]
I try to understand:

1. I need config `rangeStrategy = bump`, instead of the default value: `auto`.
2. if not, renovate only update major.

But in the repo: [react-vant](https://github.com/3lang3/react-vant), renovate create a pull request update `eslint 8.21.0 -> 8.38.0`
The link: [#563](https://github.com/3lang3/react-vant/pull/563), The only difference is that I used the group.
